app/v1/models/requestData/DesTaxYear.scala

```diff
@@ -33,16 +33,14 @@ object DesTaxYear {
   val startOfYear = 2
   val startYearAndDash = 5
 
-  /**
-    * @param taxYear tax year in MTD format (e.g. 2017-18)
-    */
-
+  //TODO REMOVE METHODS AS MTD & DES WILL USE THE SAME FORMAT
```

Comment:
If we remove the fromMtd method then the DeleteValidation for checking the MinTaxYear will need to be changed as it relies on converting to YYYY. I wonder if we should provide methods to just present and receive both date formats and this just store it as YYYY-YY?